server/src/main/resources/org/elasticsearch/common/reference-docs-links.txt

```diff
@@ -1,48 +1,48 @@
+BOOTSTRAP_CHECK_PKI_REALM                                       deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-xpack-pki-realm
+BOOTSTRAP_CHECK_ROLE_MAPPINGS                                   deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-xpack-role-mappings
+BOOTSTRAP_CHECK_TLS                                             deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-tls
+BOOTSTRAP_CHECK_TOKEN_SSL                                       deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-xpack-token-ssl
+CONTACT_SUPPORT                                                 troubleshoot/elasticsearch/elasticsearch#troubleshooting-contact-support
+UNASSIGNED_SHARDS                                               troubleshoot/elasticsearch/red-yellow-cluster-status
+EXECUTABLE_JNA_TMPDIR                                           deploy-manage/deploy/self-managed/executable-jna-tmpdir
+NETWORK_THREADING_MODEL                                         reference/elasticsearch/configuration-reference/networking-settings#modules-network-threading-model
+ALLOCATION_EXPLAIN_API                                          api/doc/elasticsearch/v9/operation/operation-cluster-allocation-explain
+NETWORK_BINDING_AND_PUBLISHING                                  reference/elasticsearch/configuration-reference/networking-settings#modules-network-binding-publishing
+SNAPSHOT_REPOSITORY_ANALYSIS                                    api/doc/elasticsearch/v9/operation/operation-snapshot-repository-analyze
+S3_COMPATIBLE_REPOSITORIES                                      deploy-manage/tools/snapshot-and-restore/s3-repository#repository-s3-compatible-services
+LUCENE_MAX_DOCS_LIMIT                                           deploy-manage/production-guidance/optimize-performance/size-shards#troubleshooting-max-docs-limit
+MAX_SHARDS_PER_NODE                                             deploy-manage/production-guidance/optimize-performance/size-shards#troubleshooting-max-shards-open
+FLOOD_STAGE_WATERMARK                                           troubleshoot/elasticsearch/fix-watermark-errors
+X_OPAQUE_ID                                                     reference/elasticsearch/rest-apis/api-conventions#x-opaque-id
+FORMING_SINGLE_NODE_CLUSTERS                                    deploy-manage/distributed-architecture/discovery-cluster-formation/modules-discovery-bootstrap-cluster#modules-discovery-bootstrap-cluster-joining
+CIRCUIT_BREAKER_ERRORS                                          troubleshoot/elasticsearch/circuit-breaker-errors
+ALLOCATION_EXPLAIN_NO_COPIES                                    api/doc/elasticsearch/v9/operation/operation-cluster-allocation-explain.html
+ALLOCATION_EXPLAIN_MAX_RETRY                                    api/doc/elasticsearch/v9/operation/operation-cluster-allocation-explain.html
```

Comment:
> ALLOCATION_EXPLAIN_MAX_RETRY  

Ditto I've created https://github.com/elastic/elasticsearch-specification/pull/3961 to copy the "maximum retries" example to the API docs, but IMO it would have more room for context and explanations if it's added to https://github.com/elastic/docs-content/blob/main/troubleshoot/elasticsearch/diagnose-unassigned-shards.md instead.